src/Profile.js

```diff
@@ -52,7 +58,15 @@ export default function CreatedGrid() {
 
   return (
     <div className="board">
-      <img src={profilpic} alt="Girl in a jacket" width="150" height="150" />
+      <div className="profileCard">
+        <img className="profileImage" src={profilpic} alt="Girl in a jacket"/>
```

Comment:
Lol I like the alt, but we are also graded on code quality. Let change to: "profile picture"

Reply:
done